packages/iov-tendermint-rpc/src/requests.ts

```diff
@@ -38,6 +39,17 @@ export type Request =
   | TxSearchRequest
   | ValidatorsRequest;
 
+/**
+ * Raw values must match the tendermint event name
+ *
+ * @see https://godoc.org/github.com/tendermint/tendermint/types#pkg-constants
```

Comment:
:+1: great reference, I should add more here as well

Reply:
When writing new docs, have a quick look at https://github.com/Microsoft/tsdoc. I am not using much of it yet but starting with `/**` and a on-liner description for the cases where some words are necessary is probably a good start